packages/jsii-pacmak/lib/targets/go.ts

```diff
@@ -229,3 +231,11 @@ async function go(command: string, args: string[], options: { cwd: string }) {
   const { cwd } = options;
   return shell('go', [command, ...args], { cwd });
 }
+
+async function tryUnlink(filePath: string) {
+  if (!(await fs.pathExists(filePath))) {
+    return;
+  }
+
+  await fs.unlink(filePath);
+}
```

Comment:
If we have `fs-extra` available, you could use `remove` from it instead of implementing this...

Reply:
Done.